run_brt.sh

```diff
@@ -35,55 +35,72 @@ export BRT_MARIAN="$( realpath ${MARIAN:-$BRT_ROOT/../build} )"
 export BRT_MODELS=$BRT_ROOT/models
 export BRT_DATA=$BRT_ROOT/data
 
+BRT_CUSTOM=false
+
 
 export LD_LIBRARY_PATH="${BRT_MARIAN}:${LD_LIBRARY_PATH}"
 
-# Check if required tools are present in marian directory
-for cmd in app/service-cli app/bergamot-translator-app; do
-    if [ ! -e $BRT_MARIAN/$cmd ]; then
-        echo "Error: '$BRT_MARIAN/$cmd' not found. Do you need to compile the toolkit first?"
-        exit 1
-    fi
-done
+if [ "$BRT_CUSTOM" = false ]; then
```

Comment:
We really don't need this false/true flag, because tags can handle WASM. Just `test -f bergamot-translator-app` suffices to know that the build directory is accurate. Keeps the script simpler, no? @abhi-agg

Reply:
I think no. Because Line 45 below searches `app/service-cli` and for custom artifacts, that is not generated.

Reply:
Read the comment on tags, the test is just for whether compilation succeeded or not, so existence of bergamot-translator-app which is common across all builds is test enough.

Reply:
I am not following you here. There are 2 options:
1. Keep `BRT_CUSTOM` flag and keep the code as it is just now. Everything will work as long as correct value for BRT_CUSTOM flag is passed while invoking this script.
2. Remove `BRT_CUSTOM` flag (which means doing following additionally)
    1. Remove the checks from this script that test whether `app/service-cli` exist, and
    2. Replace all occurrences of `app/marian-decoder-new` with `app/bergamot-translator-app`